libs/pgwire/src/main/antlr/PgArray.g4

```diff
@@ -16,48 +16,43 @@ representations. E.g.:
     {\"{\\\"x\\\": 10}\", \"{\\\"y\\\": 20}\"}
 */
 
+
+
 array
-   : '{' item (',' item)* '}'
-   | '{' '}'
-   ;
+    : '{' item (',' item)* '}'
+    | '{' '}'
+    ;
 
 item
-   : string         #value
-   | NUMBER         #value
```

Comment:
Removed the number lexer rule because we use type specific parsers for each individual element anyways. All we need from the parser is to split the input into items.